### DISCUSSION TITLE
Can I create a field that displays as a button or link?

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2987] [Archive]
Can we have fields that can be displayed as links on the details or listing page? 

Easier to browser and check them this way.

**Comment by benhaynes** [comment]
That's this: https://github.com/directus/next/issues/936

And here is how it was done in v8 in case you want to try a PR:
https://github.com/directus/app/blob/master/src/interfaces/preview/input.vue

**Reply by benhaynes** [reply]
We have this now, it's the "Button Links" Interface.
